geonode/templates/geonode/geo_header.html

```diff
@@ -24,11 +24,13 @@
 
     // Do not include the layer styler on the map-detail view.
     {% if not map %}
+        {% if user.is_authenticated %}
```

Comment:
should this be when the user is not authenticated?
Since splice will remove the tool.